public/app/features/alerting/unified/components/rules/NoRulesCTA.tsx

```diff
@@ -1,56 +1,39 @@
-import { css } from '@emotion/css';
+import { EmptyState, LinkButton, Stack, TextLink } from '@grafana/ui';
 
-import { GrafanaTheme2 } from '@grafana/data';
-import { CallToActionCard, useStyles2, Stack } from '@grafana/ui';
-import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
-
-import { logInfo, LogMessages } from '../../Analytics';
 import { useRulesAccess } from '../../utils/accessControlHooks';
 
 export const NoRulesSplash = () => {
   const { canCreateGrafanaRules, canCreateCloudRules } = useRulesAccess();
-  const styles = useStyles2(getStyles);
-  if (canCreateGrafanaRules || canCreateCloudRules) {
-    return (
-      <div>
-        <p>{"You haven't created any alert rules yet"}</p>
-        <Stack gap={1}>
-          <div className={styles.newRuleCard}>
-            <EmptyListCTA
-              title=""
-              buttonIcon="bell"
-              buttonLink={'alerting/new/alerting'}
-              buttonTitle="New alert rule"
-              proTip="you can also create alert rules from existing panels and queries."
-              proTipLink="https://grafana.com/docs/"
-              proTipLinkTitle="Learn more"
-              proTipTarget="_blank"
-              onClick={() => logInfo(LogMessages.alertRuleFromScratch)}
-            />
-          </div>
+  const canCreateAnything = canCreateGrafanaRules && canCreateCloudRules;
 
-          <div className={styles.newRuleCard}>
-            <EmptyListCTA
-              title=""
-              buttonIcon="plus"
-              buttonLink={'alerting/new/recording'}
-              buttonTitle="New recording rule"
-              onClick={() => logInfo(LogMessages.recordingRuleFromScratch)}
-            />
-          </div>
-        </Stack>
-      </div>
-    );
-  }
-  return <CallToActionCard message="No rules exist yet." callToActionElement={<div />} />;
+  return (
+    <div>
+      <EmptyState
+        message={"You haven't created any rules yet"}
+        variant="call-to-action"
+        button={
+          canCreateAnything ? (
+            <Stack direction="row" alignItems="center" justifyContent="center">
+              <LinkButton variant="primary" icon="plus" size="lg" href="alerting/new/alerting">
+                New alert rule
+              </LinkButton>
+              <LinkButton variant="primary" icon="plus" size="lg" href="alerting/new/recording">
+                New recording rule
+              </LinkButton>
+            </Stack>
+          ) : null
+        }
+      >
+        <>
+          You can also define rules through file provisioning or Terraform.{' '}
```

Comment:
I feel like this should come after the buttons?

e.g. it should be more like 
"here are the buttons you can action, but also you can define rules this way as well"

Reply:
Sadly we're somewhat restricted to what the component offers; unless we're willing to jump through a few hoops :D

https://developers.grafana.com/ui/latest/index.html?path=/story/general-emptystate--basic